projects/batfish-client/src/test/java/org/batfish/client/ClientTest.java

```diff
@@ -251,26 +251,10 @@ public void testValidJsonPath() throws IOException {
       Client.validateJsonPath(mapper.readTree(validJsonPath));
    }
 
-// ratul commented out this test
-//     parseParaValue should allow strings like "I am not valid" to parse.
-//     i can't think of what that new function will not parse
-//       TODO: remove test or give it some teeth
-//   // Tests for parseParaValue method
-//   @Test
-//   public void testParseInvalidJsonContent() {
-//      String invalidJsonContent = "I am not valid";
```

Comment:
so, where this is getting in the way is people typing   
    batfish> get questionType  a="b"
to specify parameter value b (string) for a. perhaps the validation needs to be differently structured.